lib/suspenders/version.rb

```diff
@@ -1,5 +1,5 @@
 module Suspenders
-  RAILS_VERSION = "4.2.1"
+  RAILS_VERSION = "4.2.3"
```

Comment:
we want to always use the latest stable so maybe this value should be dynamic. how? `gem install rails --explain` ? probably a better way?